avaframe/com1DFA/com1DFA.py

```diff
@@ -50,7 +50,7 @@ def initialiseRun(avaDir, flagEnt, flagRes, inputf='shp'):
     # Set directories for inputs, outputs and current work
     inputDir = os.path.join(avaDir, 'Inputs')
     outputDir = os.path.join(avaDir, 'Outputs/com1DFA')
-    workDir = os.path.join(avaDir, 'Work')
+    workDir = os.path.join(avaDir, 'Work/com1DFA')
```

Comment:
This and other spots in this file: same / problem

Reply:
done